core/v0/api/core.json

```diff
@@ -3126,18 +3118,12 @@
                     "locations": {
                         "type": "array",
                         "items": {
-                            "allOf": [
-                                {
-                                    "$ref": "#/components/schemas/Location"
-                                },
-                                {
-                                    "properties": {
-                                        "time": {
-                                            "$ref": "#/components/schemas/Time"
-                                        }
-                                    }
-                                }
-                            ]
+                            "location": {
+                                "$ref": "#/components/schemas/Location"
+                            },
```

Comment:
This too is a breaking change due to current implementations. We'll have to live with this structure for now. I suggest you do the same like you did for other schema and merge the `location.id`, `location.gps`, `location.address` and `time` in one schema.

Reply:
Decided to check with Peppo.

Reply:
Decided to add optional time schema reference into location schema.

Reply:
Done.